packages/next/src/client/route-announcer.tsx

```diff
@@ -50,12 +34,36 @@ export const RouteAnnouncer = () => {
     [asPath]
   )
 
+  const [style, className]: [React.CSSProperties?, string?] = React.useMemo(() => {
```

Comment:
It would be nice, if the css could generate via build step so that way it is not included at runtime.